config/envs/aws.py

```diff
@@ -7,4 +7,6 @@
 
 @configclass
 class AwsConfig(DefaultConfig):
-    NOTIFY_API_KEY = os.environ["NOTIFY_API_KEY"]  # fail if not present
+    # fail if these confirmation email requirements are not present
+    NOTIFY_API_KEY = os.environ["NOTIFY_API_KEY"]
+    TF_CONFIRMATION_EMAIL_ADDRESS = os.environ["TF_CONFIRMATION_EMAIL_ADDRESS"]
```

Comment:
Can we add this to the manifest? Else it will fail when deployed.

Reply:
good shout